app/build.gradle

```diff
@@ -230,7 +230,8 @@ android {
     checkAllWarnings true
     warningsAsErrors true
     ignore 'MissingTranslation', //TODO stop ignoring
-        'CheckResult'
+        'CheckResult',
+        'LogConditional'
```

Comment:
Why are we ignoring `LogConditional` here?

Reply:
Well at no point thus far have we cared about logging in production.
The TODO here should really have an associated ticket where we go through and actually improve these.
Our own dedicated logging class would be nice to have.

Reply:
I'm tagging @julianharty because I have a feeling he'll find this amusing 😉

Reply:
We have a logging ticket here: https://github.com/kiwix/kiwix-android/issues/252

Can we get a ticket for this TODO? then I'd be happy with the change 👍

Reply:
https://github.com/kiwix/kiwix-android/issues/1270
This is for specifically the conditional logging, I'll make a Lint tech debt ticket

Reply:
https://github.com/kiwix/kiwix-android/issues/1271 adding the issues to it now